### ISSUE TITLE
Staging Review finding: Context and instructions are unclear when selecting benefit

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#96653]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** My Education Benefits
**Product:** 22-5490 Form
**Feature:** (Fry/DEA) Updates

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/benefit-selection](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/benefit-selection), screen reader users may not get enough context to understand the choice they're making.

Many screen reader users navigate through a form flow as efficiently as possible --- meaning, pressing [tab] to move directly between interactive elements rather than listening to the whole page. That means any instruction text that's not programmatically associated with an input isn't likely to be heard by screen reader users.

The cards on this page are probably okay, since they've got a heading structure to help make them more discoverable to screen reader users, But once you get down to the actual question prompt and context for understanding your choices, screen reader users are unlikely to get critical context.

A screen reader user is likely to hear "Select one benefit" as their prompt to select a benefit, but they're not likely to hear "Which education benefit would you like to apply for?" or the informational note "If you're the child..."

I think it's probably possible to infer what's going on (that you need to choose one, and if you're eligible for both you have to apply separately) from "Select one benefit," so I wouldn't flag this as a launch blocker. But you're asking your blind users to go above and beyond ot figure out what's going on.

## Recommended action

Explore using the `hint` prop on the [va-radio component](https://design.va.gov/storybook/?path=/docs/uswds-va-radio--docs), or possibly grouping everything in a `fieldset` with critical information placed inside the `legend` (styled however you'd like).

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
